Docs/user_guide/getting_started.rst

```diff
@@ -0,0 +1,100 @@
+
+
+Installing 3D Slicer
+--------------------
+
+To install Slicer, click `here <http://download.slicer.org/>`_.
+
+.. figure:: https://www.slicer.org/w/images/9/90/Screen_Shot_2016-06-01_at_1.21.15_PM.png
+
+  The Nightly version of 3D Slicer is updated nightly as groups of developers make changes. The Stable version of 3D 
+  Slicer is not updated nightly and is more rigorously tested.
+
+
+Once downloaded, follow the instructions to complete installation.
+
+
+Further Documentation
+---------------------
+
+If you're interested in extending your knowledge, access the User Manual. See also the `archives of the users mailing list <http://slicer-users.65878.n3.nabble.com/>`_. The archive is searchable so most answers to questions can be found there.
+
+If you're a developer looking for more information, access the Developer Manual. See also `archives of the developer's mailing list <http://slicer-devel.65872.n3.nabble.com/>`_. Similar to the Users Mailing List archive, it is searchable.
+
+
+User Interface Overview
+-----------------------
+
+3D Slicer is built on a modular architecture. The Main Application GUI is divided into six components: the Application Menu Bar, the Application Toolbar, the Module GUI Panel, the Data Probe Panel, the 2D Slice Viewers, and the 3D Viewer.
+
+This section will introduce you to the basic functions on the main application's GUI. If you require detailed information, visit this page.
+
+
+Open 3D Slicer and load your own data or download sample data to explore. Go ahead and click around the user interface.
+
+
+.. image:: https://www.slicer.org/w/images/a/a6/LoadingData-16-06-06.png
+
+.. image:: https://www.slicer.org/w/images/2/22/SampleData-06-06.png
+
+From the Welcome panel, you can load your own data or download sample data. Sample data is often useful for exploring the features of 3D Slicer if you don't have data of your own. 
+
+.. image:: https://www.slicer.org/w/images/8/88/SliceViewOptions-2016-06-01.png
+
+.. image:: https://www.slicer.org/w/images/b/b5/ModuleExtensionAccess-2016-06-01.png
+
+Click on the push pin in the top left corner of each of the Slice Viewers or the 3D Viewer to see more options. In the Slice Viewers, the horizontal bar can be used to scroll through slices or select a slice. You can explore the various options using your loaded data or downloaded sample data.
+
+Tutorials
+---------
+
+The 3D Slicer documentation has an abundance of tutorials to help you familiarize yourself with the basics of 3D Slicer and with specific modules.
+
+Try the `Welcome Tutorial <https://www.slicer.org/wiki/Documentation/4.6/Training#Slicer_Welcome_Tutorial>`_ and the `Data Loading and 3D Visualization Tutorial <https://www.slicer.org/wiki/Documentation/4.6/Training#Slicer4_Data_Loading_and_3D_Visualization>`_ to learn the basics of using 3D Slicer.
+
+* To learn about using Slicer for 3D Printing, visit this tutorial.
```

Comment:
Links will be needed here, otherwise the reader won't know what's "this". The links are there on the corresponding wiki page
https://www.slicer.org/wiki/New_users#Tutorials